app/services/external_upload_manager.rb

```diff
@@ -28,7 +28,7 @@ def self.user_banned?(user)
 
   def self.create_direct_upload(current_user:, file_name:, file_size:, upload_type:, metadata: {})
     store = store_for_upload_type(upload_type)
-    url = store.signed_url_for_temporary_upload(file_name, metadata: metadata)
```

Comment:
See https://github.com/discourse/discourse/pull/22975/commits/d0f1c43e2a7ea60ef9dab4b53e4c65a252dff3ff for why this is necessary (along with the clientside change)